src/main/scala/viper/silver/testing/SilSuite.scala

```diff
@@ -39,7 +41,9 @@ abstract class SilSuite extends AnnotationBasedTestSuite with BeforeAndAfterAllC
     * pairs. If not prefix (colon) is given, `defaultKeyPrefix` is used as the
     * prefix. Each key in `configMap` may have at least one colon.
     */
-  lazy val prefixSpecificConfigMap: Map[String, Map[String, Any]] =
+  lazy val prefixSpecificConfigMap :  Map[String, Map[String, Any]] = prefixSpecificConfigMap(this.configMap)
```

Comment:
```suggestion
  lazy val prefixSpecificConfigMap: Map[String, Map[String, Any]] = prefixSpecificConfigMap(this.configMap)
```

(fewer spaces around `:`)